openmdao/core/group.py

```diff
@@ -848,9 +848,8 @@ def solve_linear(self, dumat, drmat, vois, mode=None, precon=False):
             sol_vec, rhs_vec = drmat, dumat
 
         # Solve Jacobian, df |-> du [fwd] or du |-> df [rev]
-        rhs_buf = {}
+        rhs_buf = OrderedDict()
```

Comment:
Actually, I'm not sure if that change is actually required. It certainly won't hurt anything, but I should check if it's actually necessary.